multipledispatch/conflict.py

```diff
@@ -1,19 +1,73 @@
 from .utils import _toposort, groupby
+from .variadic import isvariadic
+
 
 class AmbiguityWarning(Warning):
     pass
 
 
 def supercedes(a, b):
```

Comment:
Should we fix this typo? It's not new to this PR but I noticed it while reviewing the tests.

Reply:
Looks like it's a legitimate spelling: https://english.stackexchange.com/questions/305034/supercede-or-supersede

I guess we have some linguistic contrarians working on multiple dispatch :)

Reply:
nice citation, my mistake.